src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/ConstructionTapeHelper.java

```diff
@@ -107,30 +105,16 @@ public static void placeConstructionTape(@NotNull AbstractBuilding building, @No
      * @param world the world.
      */
 
-    public static void placeConstructionTape(@NotNull WorkOrderBuild workOrder, @NotNull World world)
+    public static void placeConstructionTape(@NotNull WorkOrderBuildDecoration workOrder, @NotNull World world)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)